lang/en/cachestore_redis.php

```diff
@@ -29,9 +29,15 @@
 $string['server'] = 'Server';
 $string['server_help'] = 'This sets the hostname or IP address of the Redis server to use.';
 
+$string['password'] = 'Password';
+$string['password_help'] = 'This sets the password of the Redis server.';
+
 $string['test_server'] = 'Test Server';
 $string['test_server_desc'] = 'Redis server to use for testing.';
 
+$string['test_password'] = 'Test server password';
+$string['test_password_help'] = 'Redis test server password.';
```

Comment:
I think this should be `test_password_desc` and not `test_password_help` so that it matches your changes to `settings.php`.

Thanks for the PR!